### PR TITLE
sankey nodes position

### PR DESCRIPTION
**Pull request by Mahdis-z**
resolves https://github.com/plotly/documentation/issues/1589

### CODE REVIEW

File: python/sankey-diagram.md [hunk 1]
```diff
@@ -129,6 +132,28 @@ fig.update_layout(
 fig.show()
 ```
 
+### Define Node Position
+
+The following axample sets [node.x](https://plot.ly/python/reference/#sankey-node-x) and [node.y](https://plot.ly/python/reference/#sankey-node-y) to place nodes in the specified locations, except in the [snap arrangement](https://plot.ly/python/reference/#sankey-arrangement) to avoid overlapping of the nodes, therefore, an automatic snapping of elements will be set to define the padding between nodes via [nodepad](https://plot.ly/python/reference/#sankey-node-pad). The other possible arrangements are:<font color='blue'> 1)</font> perpendicular <font color='blue'>2)</font> freeform <font color='blue'>3)</font> fixed
```
Comment (emmanuelle) [inline python/sankey-diagram.md]: typo: example

File: python/sankey-diagram.md [hunk 2]
```diff
@@ -129,6 +132,28 @@ fig.update_layout(
...
+```python
+import plotly.graph_objects as go
+
+fig = go.Figure(go.Sankey(
+    arrangement = "snap",
+    node = {
+        "label": ["A", "B", "C", "D", "E", "F"],
+        "x": [0.2, 0.1, 0.5, 0.7, 0.3, 0.5],
+        "y": [0.7, 0.5, 0.2, 0.4, 0.2, 0.3],
+        'pad':10},
```
Comment (emmanuelle) [inline python/sankey-diagram.md]: Add comment before this line "pad is given in number of pixels"

File: python/sankey-diagram.md (hunk 1 above)
Comment (emmanuelle) [reply python/sankey-diagram.md]: what is the default value of `arrangement`?

Comment (antoinerg) [reply python/sankey-diagram.md]: It is `snap` https://github.com/plotly/plotly.js/blob/f78f3cd3beee04db95aafcf9498031d112c87514/src/traces/sankey/attributes.js#L69-L81

unless `node.x` and `node.y` are defined, then it's `freeform`
https://github.com/plotly/plotly.js/blob/f78f3cd3beee04db95aafcf9498031d112c87514/src/traces/sankey/defaults.js#L88-L92

Comment (emmanuelle) [reply python/sankey-diagram.md]: thanks @antoinerg !

**Comment by emmanuelle** [comment]
:dancer: thanks!

**Pull request merged**